cumulus/polkadot-parachain/src/command.rs

```diff
@@ -832,7 +911,7 @@ pub fn run() -> Result<()> {
 				// that both file paths exist, the node will exit, as the user must decide (by
 				// deleting one path) the information that they want to use as their DB.
 				let old_name = match config.chain_spec.id() {
-				     "asset-hub-polkadot" => Some("statemint"),
+					 "asset-hub-polkadot" => Some("statemint"),
 				     "asset-hub-kusama" => Some("statemine"),
 				     "asset-hub-westend" => Some("westmint"),
 				     "asset-hub-rococo" => Some("rockmine"),
```

Comment:
What happened with formatting here? Looks like spaces before :grimacing:

Reply:
I've just checked and spaces are actually throughout this whole file. Should I replace here? Seems out of scope.

I can fix this line and make an issue/separate PR instead about fixing spaces that have crept in - what do you think? Seems strange that fmt doesn't pick this up.

Reply:
Ah the change is actually that I've replaced the spaces that were in front of here with a tab character, so this IS the fixed version.

Reply:
Yeah, I'm wondering how the others ended up as spaces. If it's just a few lines, would fix it here. If it's like the whole file, then yeah separate PR.